Tests that Runtime throws exceptions after enabling domain on scripts with errors.
Enabling Runtime Domain.
{
    method : Runtime.exceptionThrown
    params : {
        exceptionDetails : {
            columnNumber : 12
            exception : {
                className : SyntaxError
                description : SyntaxError: Unexpected token ';'
                objectId : <objectId>
                preview : {
                    description : SyntaxError: Unexpected token ';'
                    overflow : false
                    properties : [
                        [0] : {
                            name : stack
                            type : string
                            value : SyntaxError: Unexpected token ';'
                        }
                        [1] : {
                            name : message
                            type : string
                            value : Unexpected token ';'
                        }
                    ]
                    subtype : error
                    type : object
                }
                subtype : error
                type : object
            }
            exceptionId : <exceptionId>
            executionContextId : <executionContextId>
            lineNumber : 1
            scriptId : <scriptId>
            text : Uncaught SyntaxError: Unexpected token ';'
            url : syntaxError.js
        }
        timestamp : <timestamp>
    }
}
{
    method : Runtime.exceptionThrown
    params : {
        exceptionDetails : {
            columnNumber : 12
            exception : {
                className : ReferenceError
                description : ReferenceError: y is not defined     at referenceError.js:2:13
                objectId : <objectId>
                preview : {
                    description : ReferenceError: y is not defined     at referenceError.js:2:13
                    overflow : false
                    properties : [
                        [0] : {
                            name : stack
                            type : string
                            value : ReferenceError: y is not defined     at referenceError.js:2:13
                        }
                        [1] : {
                            name : message
                            type : string
                            value : y is not defined
                        }
                    ]
                    subtype : error
                    type : object
                }
                subtype : error
                type : object
            }
            exceptionId : <exceptionId>
            executionContextId : <executionContextId>
            lineNumber : 1
            scriptId : <scriptId>
            text : Uncaught ReferenceError: y is not defined
            url : referenceError.js
        }
        timestamp : <timestamp>
    }
}
